README.md

```diff
@@ -0,0 +1,108 @@
+  <a href="https://www.microverse.org/">
+    <img alt="Microverse" src="https://img.shields.io/badge/-Microverse-blueviolet?style=flat-square">
+  </a>
+
+# My Budget App
+
+My Budget App is a capstone project from the fifth module of the Microverse program. With it, we learned how to build applications using the Ruby on Rails framework.
+
+Budget App is a  application that lets you create a user, log in, so that the data is private to them. In this application you will be introduced to your new transactions associated with a category and see the money spent on each category.
+
+
+Everything will be based on the ERD class diagram presented below. 
+
+</br>
+ <div align="center">
+    <img src="./images/erd_diagram.png" alt="ERD diagram" width=50% />
+    <p>Entity Relationship Diagram (ERD) of Budget App project</p>
+</div>
+
+## [Live demo](https://shrouded-sea-17784.herokuapp.com/)
+
```

Comment:
- [ ] As part of your documentation, this project requires a video recording in which this project is presented. See requirements below:
  ![image](https://user-images.githubusercontent.com/45002654/149606879-7476bcae-8937-4115-8f9e-6cddcb08099f.png)
  
  Please do this part to make this project complete. After recording the video, please include the link in the README.